.github/scripts/generate_binary_build_matrix.py

```diff
@@ -68,7 +83,7 @@ def get_nccl_submodule_version() -> str:
 def get_nccl_wheel_version() -> str:
     import re
 
-    requrements = map(str.strip, re.split("[;|]", PYTORCH_EXTRA_INSTALL_REQUIREMENTS))
+    requrements = map(str.strip, re.split("[;|]", PYTORCH_EXTRA_INSTALL_REQUIREMENTS["12.1"]))
```

Comment:
Please apply this check for both 11.8 and 12.1 . We can use same nccl ``2.19.3``

Reply:
Done